### DISCUSSION TITLE
ec2-fleet-2.6.0

### DISCUSSION THREAD

**Discussion by pdk27** [jenkinsci/ec2-fleet-plugin#372] [General]
### What's Changed
- Make planned node name unique per EC2FleetCloud(https://github.com/jenkinsci/ec2-fleet-plugin/pull/370) @pdk27 
- [Fix logs and help](https://github.com/jenkinsci/ec2-fleet-plugin/pull/369) @pdk27 
- [[Fix] Fix computation of excess workload and available capacity] (https://github.com/jenkinsci/ec2-fleet-plugin/pull/371) @pdk27 @cjerad 

### Issues Linked
https://github.com/jenkinsci/ec2-fleet-plugin/issues/322
https://github.com/jenkinsci/ec2-fleet-plugin/issues/359

<hr /><em>This discussion was created from the release <a href='https://github.com/jenkinsci/ec2-fleet-plugin/releases/tag/ec2-fleet-2.6.0'>ec2-fleet-2.6.0</a>.</em>

**Comment by pdk27** [comment]
@wosiu @mwos-sl @wallhided @mtn-boblloyd @ikaakkola Adding you all here so you can share your findings from trying this release. Looking forward to hear back!

**Comment by wosiu** [comment]
Great to hear about the work! Appreciate verbose descirptions in the PRs 🙇 
That said it seems we are also affected by: https://github.com/jenkinsci/ec2-fleet-plugin/issues/360
We saw the very same symptoms: nodes marked as `suspended` but never deleted.

So I'm afraid I need to wait until this one is covered too, before we give a shot again, because we use configuration-as-code and reloading really often, and restarting jenkins every time to apply config change is no-go for us.

**Reply by pdk27** [reply]
Thanks for the feedback @wosiu. We are working on #360.
